### DISCUSSION TITLE
Is it possible to migrate data from one DB server to another with Prisma?

### DISCUSSION THREAD

**Discussion by tessamerrill** [prisma/prisma#6365] [Q&A]
Question is in the title. Would I have to dump the DB and restore from that on the new server, or is there a better way to do it?

Thanks!

**Comment by 2color** [comment]
> Would I have to dump the DB and restore it from that on the new server, or is there a better way to do it?

Yes, that would be the recommended way to go. Especially if the size of the database is large. 

You can find guides on how to do that with [PostgreSQL](https://www.prisma.io/dataguide/postgresql/inserting-and-modifying-data/importing-and-exporting-data-in-postgresql) and [MySQL](https://www.prisma.io/dataguide/mysql/inserting-and-modifying-data/importing-and-exporting-data-in-mysql) in [Prisma's Data Guide](https://www.prisma.io/dataguide/)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
